### DISCUSSION TITLE
Migration Timestamps

### DISCUSSION THREAD

**Discussion by Hannes-Kruger** [laravel/framework#44240] [Ideas]
Hi, I am writing some microservices using Laravel and Vapor. We are doing a lot of work on ensuring compliance from a security and auditability point of view.

We ended up doing this manually, but it would be really helpful if Laravel migrations table contained timestamps of when a migration was run.

**Comment by henzeb** [comment]
You can add them manually to the table. Just add a created_at that automatically fills on create.
